tests/tasks/test_dbus.py

```diff
@@ -0,0 +1,52 @@
+# Copyright (c) 2015, Facebook, Inc.  All rights reserved.
+#
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree. An additional grant
+# of patent rights can be found in the PATENTS file in the same directory.
+#
+from sparts.tests.base import MultiTaskTestCase, Skip
+try:
+    import dbus
+    from sparts.tasks.dbus import DBusServiceTask, DBusMainLoopTask
+except ImportError:
+    raise Skip("dbus support is required to run this test")
+
+from sparts.sparts import option
+
+class TestDBusTask(DBusServiceTask):
+    BUS_NAME = 'com.github.facebook.test'
+
+
+class TestDBusSystemTask(DBusServiceTask):
+    BUS_NAME = 'com.github.facebook.systemtest'
+
+    system_bus = option(name='system_bus', default=True)
```

Comment:
You shouldn't need to specify `name=...` in your option.  It should be inferred based on the variable name (in this case, the option would be `--system-bus`)